inginious/common/tasks_problems.py

```diff
@@ -211,6 +211,7 @@ def __init__(self, problemid, content, translations, taskfs):
         super(MultipleChoiceProblem, self).__init__(problemid, content, translations, taskfs)
         self._header = content['header'] if "header" in content else ""
         self._multiple = content.get("multiple", False)
+        self._shuffle = content.get("shuffle", False)
```

Comment:
The current behaviour is that answers are shuffled. Therefore, the default should be ``True``, or the name and effect of the feature inversed.

Reply:
You're right! :)